### DISCUSSION TITLE
Choose a Plan

### DISCUSSION THREAD

**Discussion by zelda-rageux** [slint-ui/slint#4654] [Q&A]
Hi,
I'm sorry, my English level is approximate and I don't understand licensing.
I am a single developer, I work in my company without employees, my company has less than a million euros in turnover per year.
What type of license should I purchase?

**Comment by ogoffart** [comment]
I'm assuming (based on https://github.com/slint-ui/slint/discussions/4649 ) that you are targetting embedded devices, so the royalty free license don't apply.

Is it as part of a hobby project or as part of something you are doing for your work?

If you are doing a hobby project, I recommend you do it in an open source license and then you can use the GPL license without paying anything.

If this is something you are doing for work that cannot be made open source, then you should use the "Startup & Individual" license from https://slint.dev/pricing#usage-based-plans
